### PR TITLE
feat: remove Dashboard

### PR DESCRIPTION
**Pull request by juzhiyuan**
fix https://github.com/apache/apisix-docker/pull/81

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -7,10 +7,6 @@ RUN set -x \
     && apk add --no-cache --virtual .build-deps git \
     && git clone -b ${APISIX_VERSION} https://github.com/apache/apisix.git /tmp/apisix \
     && cd /tmp/apisix \
```
Comment (juzhiyuan) [inline alpine/Dockerfile:9]: cc @membphis  Not sure if this line should be removed, if yes, kindly remove it from this PR.

Comment (moonming) [reply alpine/Dockerfile:9]: we can remove it

**Comment by membphis** [comment]
please confirm we run `alpine/Dockerfile` in CI

**Comment by juzhiyuan** [comment]
> please confirm we run `alpine/Dockerfile` in CI

https://github.com/apache/apisix-docker/blob/feat-remove-dashboard/example/docker-compose-alpine.yml#L7

**Review by moonming — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**